src/instrumentation.ts

```diff
@@ -246,6 +246,10 @@ export class HyperExpressInstrumentation extends InstrumentationBase {
         const wrapPromise = (promise: Promise<unknown>) => {
           return promise
             .then(value => {
+              if (span._span && span._span._agent && span._span._agent.currentTransaction && span._span._agent.currentTransaction._result && res.statusCode) {
+                const statusCodeStartNumber = res.statusCode.toString().charAt(0);
+                span._span._agent.currentTransaction._result = `HTTP ${statusCodeStartNumber}xx`;
```

Comment:
do we need to set it here? for example _span is not something to be accessed by outside libs

Reply:
added ts-ignore